### PR TITLE
Adds Bugsnag error monitoring

### PR DESCRIPTION
**Pull request by vinnyjth**
### What does this PR do, or why is it needed?

Sets up Bugsnag Error Handling

Credentials to log in and API key can be found by asking Vincent or @IsaacHardy 

### How do I test this PR?

1. Try to sign in with an invalid username/password.
 - You should see an error show up in the bugsnag client project 
2. Change a resolver so it `throws new Error('Some error')`
3. Query that resolver
 - you should see an error show up in bugsnag server project
---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] Closes [DEV-XXX](#url)
- [x] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android if applicable
- [x] Set two relevant reviewers

## REVIEW

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by vinnyjth** [comment]
@IsaacHardy Can you link this to the proper issue number?

File: packages/apollos-church-api/config.yml [hunk 1]
```diff
@@ -218,4 +218,4 @@ ROCK_CONSTANTS:
   TIME_ZONE: 119
   LABEL: 120
   WISTIA: 125
-  S3_ASSET: 133
+  S3_ASSET: 133
```
Comment (IsaacHardy) [inline packages/apollos-church-api/config.yml:221]: I've stared at this for 30 seconds trying to determine the difference.

File: packages/newspringchurchapp/src/index.js [hunk 2]
```diff
@@ -20,10 +20,6 @@ import UserWebBrowser from './user-web-browser';
 import Onboarding from './onboarding';
 import Prayer from './prayer';
 
-// Sentry.config(
```
Comment (IsaacHardy) [inline packages/newspringchurchapp/src/index.js:23]: buh bye

**Comment by IsaacHardy** [comment]
@vinnyjth Why would I need to add a new error to a resolver to see if Bugsnag could trigger an error event? Isn't that already built into this implementation?

**Review by IsaacHardy — APPROVED** [review]

**Pull request merged**